content/docs/addons-test-utils.md

```diff
@@ -40,17 +40,17 @@ var ReactTestUtils = require('react-dom/test-utils'); // ES5 with npm
  - [`renderIntoDocument()`](#renderintodocument)
  - [`Simulate`](#simulate)
 
-## Reference {#reference}
+## संदर्भ {#reference}
 
 ### `act()` {#act}
 
-To prepare a component for assertions, wrap the code rendering it and performing updates inside an `act()` call. This makes your test run closer to how React works in the browser.
+कौम्पोनॅन्ट की पुष्टि के लिए, उसे रेंडर व अपडेट करने वाले कोड को `act()` कॉल के अंदर लपेटें। यह आपके टेस्ट को ब्राउज़र में React जैसे काम करता है, उसके करीब लाता है।
```

Comment:
कौम्पोनॅन्ट की पुष्टि के लिए is not the correct translation here

Reply:
लपेटें -> सम्मिलित kare

Reply:
करीब लाता है।
 -> kareeb laega

Reply:
Made all three changes.

Reply:
assertion could be left as is since पुष्टि does not seem right?